src/doc/bootstrap

```diff
@@ -125,11 +134,48 @@ Some packages that provide additional functionality are marked as
 integration of these packages into the Sage distribution.
 
 EOF
-for PKG_BASE in $(sage-package list --has-file SPKG.rst :experimental: | sort); do
+for PKG_BASE in $(sage-package list --has-file SPKG.rst :experimental: | grep -v '^sagemath_' | sort); do
     echo "* :ref:\`spkg_$PKG_BASE\`"
 done >> "$OUTPUT_INDEX"
 cat >> "$OUTPUT_INDEX" <<EOF
 
+Runtime-Detectable Features and Conditional Doctests
+----------------------------------------------------
+
+.. toctree::
+   :maxdepth: 1
+
+   sage/features
+   sage/features/sagemath
+   sage/features/pkg_systems
+   sage/features/bliss
+   sage/features/csdp
+   sage/features/databases
+   sage/features/dvipng
+   sage/features/ffmpeg
+   sage/features/four_ti_2
+   sage/features/gap
+   sage/features/graph_generators
+   sage/features/graphviz
+   sage/features/imagemagick
+   sage/features/interfaces
+   sage/features/internet
+   sage/features/kenzo
+   sage/features/latex
+   sage/features/latte
+   sage/features/lrs
+   sage/features/mcqd
+   sage/features/meataxe
+   sage/features/mip_backends
+   sage/features/normaliz
+   sage/features/pandoc
+   sage/features/pdf2svg
+   sage/features/polymake
+   sage/features/rubiks
+   sage/features/tdlib
+   sage/features/join_feature
+   sage/features/all
+
```

Comment:
How about splitting feature facilities from features themselves? So `sage/features`, `sage/features/join_feature`, `sage/features/all` may be placed fore-front and then features afterwards.

Reply:
In a separate toctree then?

Reply:
Maybe. Perhaps we may later move them three out of this section if we could find a better place for them. But I have no idea right now.

Reply:
Good place for the three may be 

Infrastructure > Programming Facilities > Test Framework

or (new)

Infrastructure > Programming Facilities > Features Framework

Reply:
Right, that's where they are coming from...